data-pipeline/terraform/container_apps.tf

```diff
@@ -14,136 +14,50 @@ resource "azurerm_container_app_environment" "main" {
   tags = local.common-tags
 }
 
-# resource "azurerm_user_assigned_identity" "container-app" {
-#   location            = azurerm_resource_group.resource-group.location
-#   name                = "${var.environment-prefix}containerappmi"
-#   resource_group_name = azurerm_resource_group.resource-group.name
-# }
-#
-# resource "azurerm_role_assignment" "container-app" {
-#   scope                = data.azurerm_container_registry.acr.id
-#   role_definition_name = "acrpull"
-#   principal_id         = azurerm_user_assigned_identity.container-app.principal_id
-#   depends_on = [
-#     azurerm_user_assigned_identity.container-app
-#   ]
-# }
-
-resource "azurerm_container_app" "data-pipeline" {
-  name                         = "${var.environment-prefix}-ebis-data-pipeline"
-  container_app_environment_id = azurerm_container_app_environment.main.id
-  resource_group_name          = azurerm_resource_group.resource-group.name
-  revision_mode                = "Single"
-  workload_profile_name        = "Pipeline"
-
-  identity {
-    type = "SystemAssigned"
-    #     identity_ids = [azurerm_user_assigned_identity.container-app.id]
-  }
+module "container_app_default" {
+  source = "./container_app"
 
-  secret {
-    name  = "queue-connection-string"
-    value = data.azurerm_storage_account.main.primary_connection_string
-  }
+  container-app-environment-id = azurerm_container_app_environment.main.id
```

Comment:
Need to clarify expectations between `azurerm_container_app_environment` and `azurerm_container_app` in the context of the new Apps. running: should each Container App share the same Container App Environment? Or each have a unique one?

Reply:
Discussed with @jrabbott and will leave this as-is for now.